Makefile

```diff
@@ -200,6 +200,10 @@ fmt: format
 lint:
 	if [ $$(wc -l < tests/utils.go) -gt 2813 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
 
+	# CGO_ENABLED=0 is needed to avoid typecheck isseus with cmd/container-disk-v2alpha/main.c
+	# The error is on pre-processing whereas skip-files and skip-dirs are post-processing
```

Comment:
>	# The error is on pre-processing whereas skip-files and skip-dirs are post-processing

Did we open an issue about this?

Reply:
IIRC, this is a `golangci-lint` problem itself or `forbidigo` one. The issue is related to how pre-processing the directories and files work and how that's passed to linters. For example, I tried to set `--skip-dirs`  and `--skip-files` to avoid that path/file and it still failed. The `CGO_ENABLED=0` was the workaround suggestion.